aiida/cmdline/commands/daemon.py

```diff
@@ -261,24 +261,6 @@ def daemon_stop(self, *args, **kwargs):
                 if pid is None:
                     dead = True
                     print "AiiDA Daemon shut down correctly."
```

Comment:
To achieve the goal, shouldn't you also remove the load_dbenv call in the three lines 234-236 above? (as well as the import of get_last_daemon_timestamp etc)

Reply:
Of course, weird I could have sworn I removed it but something must have gone in saving the file. Anyway, I corrected it. Good catch!